design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -199,6 +209,17 @@ input NegotiableQuoteFilterInput {
     name: FilterMatchTypeInput @doc(description: "Filter by negotiable quote name")
 }
 
+input NegotiableQuoteSortInput {
+    name: SortEnum @doc(description: "Sort by negotiable quote name")
```

Comment:
This looks good. Additiona note, Luma doesn't support sorting at this point. 
<img width="1033" alt="Screen Shot 2021-04-09 at 1 45 31 PM" src="https://user-images.githubusercontent.com/43612864/114227029-0c648800-993a-11eb-98d7-2c52a06a9be3.png">